openlibrary/plugins/openlibrary/js/bookreader_direct.js

```diff
@@ -0,0 +1,20 @@
+// Make Borrow links act as if POSTing to Borrow page
+
+jQuery(function() {
+  // TODO: After update jQuery, will need to get rid of deprecated .live()
+  $('.borrow-link').live('click', function(event) {
```

Comment:
agreed, it would be preferable to do something like:

$(body).on("click", ".borrow-link", function() { ... });

to attach the listener after the element is added to the dom / live

This is only available as of 1.7+, see http://api.jquery.com/live/